3.6/debian-9/rootfs/opt/bitnami/scripts/mongodb-env.sh

```diff
@@ -39,6 +39,7 @@ mongodb_env_vars=(
     MONGODB_USERNAME
     MONGODB_PASSWORD
     MONGODB_DATABASE
+    MONGODB_EXTRA_DATABASES
```

Comment:
You can drop this. Instead let's declare the new env. vars here (keeping the old ones for backwards compatibility):
```suggestion
    MONGODB_USERNAMES
    MONGODB_PASSWORDS
    MONGODB_DATABASES
```

Reply:
Good catch for the `MONGODB_EXTRA_DATABASES` removal!

As for keeping the old variables there, I ... don't think that they should be there if we opt to align ourselves on other Bitnami's images implementations. For example, the old variable `KAFKA_SEGMENT_BYTES` does not appear [here](https://github.com/bitnami/bitnami-docker-kafka/blob/master/2/debian-10/rootfs/opt/bitnami/scripts/kafka-env.sh) but is still aliased and referenced from [here](https://github.com/bitnami/bitnami-docker-kafka/blob/9d92b711132bb42e133f6d88e124301070c2fd12/2/debian-10/rootfs/opt/bitnami/scripts/libkafka.sh#L187)

Reply:
As I mentioned in my previous comment, we're standardizing to use this file to manage everything related to the declaration of the env. variables and assigning its default values.